code/_onclick/click.dm

```diff
@@ -71,7 +71,11 @@
 		return
 
 	if(istype(loc,/obj/mecha))
-		if(!locate(/turf) in list(A,A.loc)) // Prevents inventory from being drilled
+		// Prevents inventory from being drilled
+		// Allows using inventory buttons
+		if(!locate(/turf) in list(A,A.loc) || istype(A, /obj/screen))
+			var/obj/item/W = get_active_hand()
+			ClickInventory(A, W, params)
```

Comment:
I put that there so that inventory actions could be used in a mech that specifically had inventory enabled. This will normally not do anything, since inventory in a mech is disabled, but when inventory in a mech *is* enabled, one will be able to interact with items in your inventory.